lua/catppuccin/groups/integrations/feline.lua

```diff
@@ -292,7 +292,8 @@ function M.get()
 	-- workspace loader
 	components.active[2][1] = {
 		provider = function()
-			local Lsp = vim.lsp.util.get_progress_messages()[1]
+			-- local Lsp = vim.lsp.status()[1]
+			local Lsp = vim.fn.has "neovim-0.10" == 1 and vim.lsp.util.get_progress_messages()[1] or vim.lsp.status()[1]
```

Comment:
Uhh, this returns 0, so this breaks for neovim-stable users

Reply:
Yeah, this errors for me on 0.9.1, this works:
```suggestion
			local Lsp
			if vim.fn.has "neovim-0.10" then
				Lsp = vim.lsp.util.get_progress_messages()[1]
			else
				Lsp = vim.lsp.status()[1]
			end
```